packages/block-library/src/file/edit.js

```diff
@@ -102,7 +102,11 @@ function FileEdit( { attributes, isSelected, setAttributes, clientId } ) {
 		}
 
 		if ( downloadButtonText === undefined ) {
-			changeDownloadButtonText( _x( 'Download', 'button label' ) );
+			setAttributes( {
+				downloadButtonText: removeAnchorTag(
+					_x( 'Download', 'button label' )
+				),
```

Comment:
Although this would not be necessary in practice, it allows for the possibility that the translated text has anchors.

Reply:
This translated text shouldn't have HTML in it, and React shouldn't render it anyway. So I agree this is unnecessary.

We can't strip every possible translation.

Reply:
Indeed. Updated with 0bf95cd 👍